mlir/lib/Target/LLVMIR/ModuleImport.cpp

```diff
@@ -2201,6 +2201,10 @@ LogicalResult ModuleImport::convertInstruction(llvm::Instruction *inst) {
                 builder.getStringAttr(asmI->getAsmString()),
                 builder.getStringAttr(asmI->getConstraintString()),
                 asmI->hasSideEffects(), asmI->isAlignStack(),
+                callInst->isTailCall()
+                    ? TailCallKindAttr::get(mlirModule.getContext(),
+                                            TailCallKind::Tail)
```

Comment:
This implies that there is only one supported kind. If so, would it make sense to model this simply as a unit attribute instead of supporting the enum but only one value of it?

Reply:
This was a mistake on my side, I made the translation generic and forgot to patch the importer. `notail` is also accepted.

Reply:
Fixed.